src/software/thunderscope/replay/replay_file_debugging_script.py

```diff
@@ -0,0 +1,75 @@
+import base64
+import glob
+import gzip
+import os
+import argparse
+
+from proto.import_all_protos import *
+from software.thunderscope.replay.replay_constants import *
+
+if __name__ == "__main__":
```

Comment:
sometimes this script gets stuck for some reason

Reply:
[proto_2024_06_17_214626.zip](https://github.com/user-attachments/files/15880627/proto_2024_06_17_214626.zip)

Reply:
let me investigate.

Reply:
I think I've fixed it. Could you check?

Reply:
I tested the same replay file on my end and it seems good. There's just a caught error at the end: `EOFError: Compressed file ended before the end-of-stream marker was reached` which is part of the last chunk, so it's probably because Arun abruptly closed proto logger.